modules/bvl_feedback/ajax/react_get_bvl_threads.php

```diff
@@ -2,46 +2,43 @@
 /**
  * Used by react on initial load to load the thread state.
  *
- * PHP version 5
+ * PHP version 7
  *
  * @category Behavioural
  * @package  Main
  * @author   Evan McIlroy <evanmcilroy@gmail.com>
  * @license  GPLv3 <http://www.gnu.org/licenses/gpl-3.0.en.html>
  * @link     https://www.github.com/aces/Loris-Trunk/
  */
-header("content-type:application/json");
-ini_set('default_charset', 'utf-8');
```

Comment:
This seems to be changing behaviour outside of what's required for CandID

Reply:
Yeah okay I can revert these.